templates/admin/user/index.html.ep

```diff
@@ -4,7 +4,20 @@
 % content_for 'ready_function' => begin
   $('#users').DataTable( {
     "order": [[0, 'asc']],
-} );
+  } );
+
+$('input[name="role"]:radio').change(function() {
+   var username = $(this).parents('tr').find('.name').text();
+   var role = $(this).attr('id');
+   role = $('label[for="' + role + '"]').text();
+   if (confirm("Sure to make " + username + " a " + $.trim(role) + "?")) {
```

Comment:
I would like it to be more formal. "Are you sure to ....". But my English is what it is so maybe I'm misguided.